Demo/Common/Networking/SessionRequest.swift

```diff
@@ -46,6 +47,8 @@ internal struct SessionRequest: APIRequest {
         //        try container.encode(installmentOptions, forKey: .installmentOptions)
 
         if ConfigurationConstants.current.cardComponentConfiguration.showsStorePaymentMethodField {
+            AdyenAssertion.assert(message: "API version should be v70 or above to apply card component's store payment method field",
```

Comment:
Maybe just `if` and popup? 🤔

Reply:
We discussed this last week and this was the conclusion but we can discuss more if you want 😄

Reply:
I like that the assert gives you more context about what/where the issue is here by "teleporting" you into the right part of the code which encourages exploration of the sample code.

Reply:
We can discuss this further and change if needed in another PR.